utils/merkletrie/filesystem/node.go

```diff
@@ -0,0 +1,128 @@
+package filesystem
```

Comment:
this should probably be moved to the billy package, but definately it should not be under merkletrie utilities.